brokers/brokers.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package brokers
```

Comment:
You should consider changing name, not be plural. There are exceptions like `strings` `bytes` etc... but in general, should be singular.